datafusion/src/datasource/file_format/parquet.rs

```diff
@@ -419,7 +423,8 @@ mod tests {
 
     #[tokio::test]
     async fn read_alltypes_plain_parquet() -> Result<()> {
-        let runtime = Arc::new(RuntimeEnv::default());
+        let session_ctx = SessionContext::new();
+        let task_ctx = Arc::new(TaskContext::from(&session_ctx));
```

Comment:
> Since this is such a common pattern (create a Arc` I wonder if it would make sense to create a function for it?
> 
> like
> 
> ```rust
> impl SessionContext {
>   /// Get a new TaskContext to run in this session
>   pub fn task_ctx(&self) -> Arc<TaskContext> {
>     ...
>   }
> }
> ```

Done.